ufl/sobolevspace.py

```diff
@@ -179,5 +178,6 @@ def __str__(self):
 HCurl = SobolevSpace("HCurl", [L2])
 H1 = SobolevSpace("H1", [HDiv, HCurl, L2])
 H2 = SobolevSpace("H2", [H1, HDiv, HCurl, L2])
+HInf = SobolevSpace("HInf", [H2, H1, HDiv, HCurl, L2])
```

Comment:
While we're here, can we update the parent list for all of these to include the `HEin` and `HDivDiv` spaces below, I think that they have stronger continuity requirements than `HCurl` and `HDiv` respectively. Although maybe one doesn't want that since really `HDivDiv in Vector(HDiv)` since the former is tensor fields and the latter vector fields.

Reply:
Let's pass on this one.

Reply:
Do you want to open an issue for this?